diff --git a/activity/activity.info b/activity/activity.info
index 287bce9..2e18b56 100644
--- a/activity/activity.info
+++ b/activity/activity.info
@@ -1,8 +1,16 @@
 [Activity]
-name = Tic-Tac-Toe
+name = TicTacToe
 activity_version = 3
 bundle_id = org.sugarlabs.TicTacToe
 exec = sugar-activity3 activity.TicTacToe
 icon = activity-tictactoe
 license = GPLv3+
-repository = https://github.com/devmrfitz/tictactoe
+repository = https://github.com/sugarlabs/tictactoe
+release_date = 2024-03-04
+metadata_license = CC0-1.0
+summary = A TicTacToe game application.
+description = This is a TicTacToe (OX) game where two players can play.
+url = https://github.com/sugarlabs/tictactoe.git
+tags = Game
+update_contact = shubhamtiwari71488@gmail.com
+developer_name = Shubham Tiwari
\ No newline at end of file
